hbase-server/src/main/java/org/apache/hadoop/hbase/regionserver/HRegionServer.java

```diff
@@ -3906,7 +3906,7 @@ public boolean isShutDown() {
   /**
    * Force to terminate region server when abort timeout.
    */
-  private static class SystemExitWhenAbortTimeout extends TimerTask {
+  public static class SystemExitWhenAbortTimeout extends TimerTask {
```

Comment:
It is reasonable to keep it private since it is an inner task which should not be seen outside service.

I suggest using 
`getDeclaredConstructor().setAccessible(true).newInstance()` to bypass it.

Reply:
ok, good suggest, setAccessible return void, so should write like this
```
    Constructor<? extends TimerTask> taskClazz =
      Class.forName(TestGetDelaredConstructor.SystemExitWhenAbortTimeout.class.getName())
        .asSubclass(TimerTask.class).getDeclaredConstructor();
    taskClazz.setAccessible(true);
    return taskClazz.newInstance();
```

Reply:
@Reidddddd 
done